src/overlays/actors/ovl_Bg_Gnd_Darkmeiro/z_bg_gnd_darkmeiro.h

```diff
@@ -10,10 +10,29 @@ typedef void (*BgGndDarkmeiroActionFunc)(struct BgGndDarkmeiro*, GlobalContext*)
 
 typedef struct BgGndDarkmeiro {
     /* 0x0000 */ DynaPolyActor dyna;
-    /* 0x0164 */ char unk_164[0x8];
+    /* 0x0164 */ u16 actionState;    /* Bitflags for whether a block or timer is on.
+                                        0x8 for timer 2, 0x4 for timer 1, 0x2 for blocks.   */
```

Comment:
code looks good to me.
just not sure on the name of this var. at the very least it should have the word flags in it imo

and action state sounds redundant. do the bit flags actually correspond to the action func?
a common pattern for this is to call it behavior if its separate from the action funcs. But if its directly tied, then actionFlags is probably good

Reply:
I like actionFlags better than behavior. The flags keep track of what it's doing right now (counting down one or more timers or being on or off).

Reply:
do they correspond to the action function though? if they dont then they shouldnt have conflicting names